nubis/puppet/salesforce_sfmc.pp

```diff
@@ -0,0 +1,118 @@
+cron::daily { "${project_name}-salesforce_sfmc":
+  user    => 'etl',
+  command => "nubis-cron ${project_name}-salesforce_sfmc /opt/etl/salesforce_sfmc/run",
+}
+
+python::pyvenv { "${virtualenv_path}/sfmc-fetcher" :
+  ensure  => present,
+  version => '3.4',
+  require => [
+    File[$virtualenv_path],
+  ],
+}
+
+python::pyvenv { "${virtualenv_path}/vertica-csv-loader" :
+  ensure  => present,
+  version => '3.4',
+  require => [
+    File[$virtualenv_path],
+  ],
+}
+
+# Install Mozilla's sfmc-fetcher
+python::pip { 'sfmc-fetcher':
+  ensure     => 'present',
+  virtualenv => "${virtualenv_path}/data-integrations",
+  #url        => 'git+https://github.com/mozilla-it/data-integrations@f14872bb716f21ca965d8c6f7dbf12cc27147db1',
+  require    => [
+  ],
+}
+
+file { '/usr/local/bin/sfmc-fetcher':
+  ensure  => link,
+  target  => '/usr/local/virtualenvs/data-integrations/bin/brickftp_poc.py',
+  require => [
+    Python::Pip['sfmc-fetcher'],
+  ],
+}
+
+# Install Mozilla's vertica-csv-loader
```

Comment:
Puppet doesn't support duplicated, this isn't needed, as salesforce.pp already takes care of this.